src/System.IO.Compression/ref/4.0.0/System.IO.Compression.depproj

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <OutputType>Library</OutputType>
+    <PackageTargetFramework>dotnet5.2</PackageTargetFramework>
```

Comment:
@akoeplinger I don't know if we have a specific ETA on documenting this stuff yet but we are fully aware there is a lot of confusion around and we do plan on trying to address that confusion.